### ISSUE TITLE
OptionList accessibility doesn't work correctly?

### ISSUE THREAD

**Issue by krailler** [react-component/select#473]
Hello :)

I saw on the code, that the OptionList renders a "listbox" role for accessibility, and that function calls renderItem (https://github.com/react-component/select/blob/master/src/OptionList.tsx#L230), but, the value always is the prop "value", and this prop doesn't contain the real description. I saw also that the list use "mergedLabel" (https://github.com/react-component/select/blob/master/src/OptionList.tsx#L291), conditional with childrenAsData to use the prop "children".

Why the renderItem for the listbox is not using that?

Changing the function like this works perfectly.
```diff
  function renderItem(index) {
    var item = memoFlattenOptions[index];
    var value = item && item.data.value;
+   const mergedLabel = childrenAsData ? item && item.data.children : item && item.data.label;
    return item ? React.createElement("div", {
      key: index,
      role: "option",
      id: "".concat(id, "_list_").concat(index),
      "aria-selected": values.has(value)
-   }, value) : null;
+   }, mergedLabel || value) : null;
  }
```

Any thoughts?
Thanks,
